### PR TITLE
Change validation ID to URI

### PR DESCRIPTION
**Pull request by natifridman**
Change validation ID to URI from https://github.com/redhat-partner-solutions/vse-sync-test/

### CODE REVIEW

File: pkg/vaildations/cluster_version.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ import (
 )
 
 const (
-	clusterVersionID  = "Opensift Cluster Version is valid"
+	clusterVersionID  = "https://github.com/redhat-partner-solutions/vse-sync-test/tree/main/tests/sync/G.8272/environment/version/ocp/testspec.adoc"
```
Comment (spenced) [inline pkg/vaildations/cluster_version.go:20]:
```suggestion
	clusterVersionID  = "https://github.com/redhat-partner-solutions/vse-sync-test/tree/main/tests/sync/G.8272/environment/version/ocp/"
```

Comment (spenced) [reply pkg/vaildations/cluster_version.go:20]: testspec.adoc isn't (formally) part of the test URI

Comment (natifridman) [reply pkg/vaildations/cluster_version.go]: Correct, removed

**Review by jnunyez — APPROVED** [review]
lgtm

File: pkg/vaildations/cluster_version.go [hunk 2]
```diff
@@ -17,7 +17,7 @@ import (
 )
 
 const (
-	clusterVersionID  = "Opensift Cluster Version is valid"
+	clusterVersionID  = "https://github.com/redhat-partner-solutions/vse-sync-test/tree/main/tests/sync/G.8272/environment/version/ocp/"
```
Comment (nocturnalastro) [inline pkg/vaildations/cluster_version.go:20]: You could put the `https://github.com/redhat-partner-solutions/vse-sync-test/tree/main/tests/sync/G.8272/environment` into a const in common will make the line shorter,

**Review by nocturnalastro — CHANGES_REQUESTED** [review]
Just a small tweak so we don't have huge lines

File: pkg/vaildations/cluster_version.go (hunk 2 above)
Comment (nocturnalastro) [reply pkg/vaildations/cluster_version.go:20]: This is what im suggesting :
```go 
// in common.go
const TGMEnvBaseURI = "https://github.com/redhat-partner-solutions/vse-sync-test/tree/main/tests/sync/G.8272/environment/"

// in cluster_version.go
const clusterVersionID = TGMEnvBaseURI + "version/ocp/"
```

If we have another base path add another constant. 

Depends what you mean by readable.

It is harder to look at `cluster_version.go` and see exactly what `clusterVersionID` is going to be. 

However from a I'm working in the code perspective I think this is loads more readable as it removes stuff that isn't really relevant and replaces it with a summary that actually means something.

Comment (natifridman) [reply pkg/vaildations/cluster_version.go]: Done

**Comment by nocturnalastro** [comment]
@natifridman Can you squash any relevant commits then merge when your happy :)

**Comment by nocturnalastro** [comment]
@natifridman Try now :)

**Pull request merged**